DQM/BeamMonitor/plugins/FakeBeamMonitor.cc

```diff
@@ -903,6 +910,12 @@ void FakeBeamMonitor::FitAndFill(const LuminosityBlock& lumiSeg, int& lastlumi,
   //    }
   //  }
   //  theBeamFitter->SetPVInfo(DipPVInfo_);
+  DipPVInfo_.push_back(rndm_->Gaus(1000., 100.));  // Events used
+  DipPVInfo_.push_back(rndm_->Gaus(100., 10.));    // Mean PV
+  DipPVInfo_.push_back(rndm_->Gaus(10., 5.));      // Mean PV err
+  DipPVInfo_.push_back(rndm_->Gaus(10., 5.));      // Rms PV
+  DipPVInfo_.push_back(rndm_->Gaus(5., 3.));       // Rms PV err
+  DipPVInfo_.push_back(rndm_->Gaus(100., 10.));    // Max PVs
   countEvtLastNLS_ = 0;
```

Comment:
based on https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-bbe8e0/18747/llvm-analysis/report-5cb7d6.html#EndPath
"Value stored to 'countEvtLastNLS_' is never read"

Reply:
That is because in `FakeBeamMonitor.cc` the BeamSpot does not come from an actual fit of the events, it's filled with random values, and the same is for the quantities in `DipPVInfo_`, so that counter is actually never used (contrary to `BeamMonitor.cc` where it is is actually used).